imagemodal/views.py

```diff
@@ -22,13 +21,22 @@ class ImageModalViewMixin(
     """
 
     loader = ResourceLoader(__name__)
+    static_css_extra = [
+        URL_FONT_AWESOME_CSS,
+        'view.css',
+    ]
+    static_js = [
+        'draggabilly.pkgd.min.js',
```

Comment:
Does this work with the imported version from google? (Or wherever it's coming from)

Reply:
I don't follow... Google?

Reply:
Didn't we stop loading that file locally and use an online version?

Reply:
> Didn't we stop loading that file locally and use an online version?

No.

I'm concerned that you approved this PR that refactors how we build fragments and pull in static assets, but don't follow how we build fragments and pull in static assets.
It's alright if you don't understand, but please don't approve code you don't follow.

I'd like for you to spend a little while going over this code and tracing the execution.
After you've spent some time trying to grok it, we can sit down and you can walk me through how you think it works and I can fill in any blanks.

Reply:
Ah nvmd, we're importing this from a node_module. I got confused for a second.